src/main/java/com/microsoft/aad/msal4j/RequestContext.java

```diff
@@ -44,6 +45,14 @@ public RequestContext(AbstractClientApplicationBase clientApplication,
         this.apiParameters = apiParameters;
     }
 
+    public RequestContext(AbstractClientApplicationBase clientApplication,
+                          PublicApi publicApi,
+                          IAcquireTokenParameters apiParameters,
```

Comment:
userIdentifier should be part of IAcquireTokenParameters, so should be available through it, so no need to add this constructor ?

Reply:
`UserIdentifier` is not part of `IAcquireTokenParameters`, because not all acquire token calls will have an `UserIdentifier` (for example, client credentials).